docs/no_docker_mode.md

```diff
@@ -0,0 +1,81 @@
+Armory No-Docker Mode 
+=======================
+In order to run armory in `--no-docker` mode, you will need a properly
+setup environment.  Generally folks have used conda in the past, however this
+document only requires a python (>=3.7, <3.9) environment to get going.
+
+First you will need to clone the armory repo (or if you plan to be a developer, 
+see [Contributing to Armory](./contributing.md) to clone a fork of the repo).
+For the following, the repo directory will be referred to as `[armory-repo]`.
+
+Now you will want to create a clean python virtual environment and activate
+that environment.  For example:
+```bash
+cd [armory-repo]
+python37 -m venv venv37
+source venv37/bin/activate
+```
+or you can setup your virtualenv using another tool (e.g. conda, pyenv, etc.).
+
+Once this is complete, and you have ensured you are in the `[armory-repo]` directory, 
+you can setup the environment with the following:
+```bash
+pip install --upgrade pip==22.0.3
+pip install -r no-docker-requirements.txt
+pip install -e . 
+```
+Once this completes, you should run `armory configure` to setup the armory configuration 
+(e.g. dataset download directory, output directory, etc.).
+
+With this complete, you now can run armory using `armory run -h`.  If you would 
+like to test the installation / environment, we have provided some base tests that
+can be executed using:
+```bash
+pytest -s ./tests/unit/test_no_docker.py
+```
+
+This runs a series of configs in a variety of ways to ensure that 
+the environment is operating as expected. 
+
+NOTE: If you run into issues running pytest (e.g. sometimes your `$PATH` is configured
+to point to a global pytest that is outside your virtualenv) directly, you can use the 
+alternative approach (make sure your virtualenv is active):
+```bash
+python -m pytest -s ./tests/unit/test_no_docker.py
+```
+
+If you would like to run
+the example interactively you can use:
+```bash
+pytest -s ./tests/unit/test_no_docker.py::test_interactive
+```
+or you can try yourself by entering python session and typing:
+```python
+from armory import scenarios
+from armory import paths
+paths.set_mode("host")
+
+config = os.path.join(".","scenario_configs","no_docker","cifar_short.json")
```

Comment:
This needs an `import os` prior to calling this.

Alternatively, using the new pathlib, you could do:
```
from pathlib import Path
config = Path("scenario_configs/no_docker/cifar_short.json")
```

Reply:
updated.